### PR TITLE
ndmp: add more log messages for errors

### PR DESCRIPTION
**Pull request by franku**

### CODE REVIEW

File: core/src/dird/ndmp_dma_storage.cc [hunk 1]
```diff
@@ -301,36 +303,53 @@ static bool NdmpRunStorageJob(JobControlRecord* jcr,
...
+    Dmsg0(200, "Could not validate ndma job.\n");
+    return false;
+  }
 
   /*
    * Commission the session for a run.
    */
-  if (ndma_session_commission(ndmp_sess)) { goto bail_out; }
+  if (ndma_session_commission(ndmp_sess)) {
+    Dmsg0(200, "Could not commission the ndma sessiond.\n");
```
Comment (pstorz) [inline core/src/dird/ndmp_dma_storage.cc]: Is the "d" in sessiond an typo?

File: core/src/dird/ndmp_dma_storage.cc [hunk 2]
```diff
@@ -634,7 +666,10 @@ bool NdmpUpdateStorageMappings(UaContext* ua, StorageResource* store)
 {
   struct ndm_session* ndmp_sess;
 
-  if (!GetRobotElementStatus(ua->jcr, store, &ndmp_sess)) { return false; }
+  if (!GetRobotElementStatus(ua->jcr, store, &ndmp_sess)) {
+    Dmsg0(200, "Could not get GetRobotElementStatus.\n");
```
Comment (pstorz) [inline core/src/dird/ndmp_dma_storage.cc]: I think this should be identical to the wording in line 651:
Dmsg0(200, "Could not get robot element status.\n");

File: core/src/dird/ndmp_dma_storage.cc [hunk 3]
```diff
@@ -669,7 +707,10 @@ drive_number_t NdmpGetNumDrives(UaContext* ua, StorageResource* store)
   /*
    * See if the mappings are already determined.
    */
-  if (!NdmpUpdateStorageMappings(ua, store)) { return drives; }
+  if (!NdmpUpdateStorageMappings(ua, store)) {
+    Dmsg0(200, "NdmpUpdateStorageMappings failed\n");
```
Comment (pstorz) [inline core/src/dird/ndmp_dma_storage.cc]: Maybe the Dmsg should contain a prefix of "NdmpGetNumSlots:" and "NdmpGetNumDrives:" here and lin line 692

Comment (franku) [reply core/src/dird/ndmp_dma_storage.cc]: Dmsg0 prints out __FILE__ and __LINE__

**Pull request merged**